media/css/firefox/family/index.less

```diff
@@ -0,0 +1,886 @@
+    margin: 0 auto;
+
+    @media only screen and (max-width: @breakDesktopWide) {
+        width: @widthDesktop;
+    }
+
+    @media only screen and (max-width: @breakDesktop) {
+        width: @widthTablet;
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        width: @widthMobile;
+    }
+
+    @media only screen and (min-width: @breakMobileLandscape) and (max-width: @breakTablet) {
+        width: @widthMobileLandscape;
+    }
+}
+
+
+// Handy sprite mixin
+.fxfamily-sprite(@posx, @posy) {
+    background-image: url(/media/img/firefox/family/index/family-sprite.png);
+    background-position: @posx @posy;
+    background-repeat: no-repeat;
+    overflow: hidden;
+    position: absolute;
+    display: block;
+    content: '';
+    .at2x('/media/img/firefox/family/index/family-sprite.png', 500px, 645px);
+}
+
+
+
+/*--------------------------------------------------------------------------*/
+// Conditional download bar
+#conditional-download-bar {
+    background: #fff;
+    color: @textColorPrimary;
+    min-height: 1.5em;
+    overflow: hidden;
+    padding: 15px 0;
+    text-align: center;
+    width: 100%;
+    .transition(all 0.25s ease);
+
+    .message {
+        display: none;
+        padding: 0 (@gridGutterWidth * 3) 0 @gridGutterWidth;
+    }
+
+    .btn-download {
+        .font-size(@smallFontSize);
+        padding: .5em 30px;
+        margin: 5px 10px 0;
+        background-color: #6fbe4a;
+
+        &:hover,
+        &:focus,
+        &:active {
+            background-color: lighten(#6fbe4a, 5%);
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            display: block;
+        }
+    }
+
+    .btn-close {
+        .image-replaced();
+        border: 0;
+        cursor: pointer;
+        height: 30px;
+        width: 30px;
+        padding: 0;
+        position: absolute;
+        right: 30px;
+        top: 10px;
+        background: #efefef url(/media/img/firefox/family/index/closex.png) center center no-repeat;
+        .at2x('/media/img/firefox/family/index/closex.png', 19px, 19px);
+
+        &:hover {
+            background-color: darken(#efefef, 5%);
+        }
+    }
+}
+
+#conditional-download-bar #dlbar-nonfx {
+    display: block;
+}
+
+html.firefox-latest {
+    #conditional-download-bar  {
+        display: none;
+    }
+}
+
+html.firefox-old {
+    #conditional-download-bar #dlbar-oldfx-desktop {
+        display: block;
+    }
+}
+
+html.ios {
+    #conditional-download-bar #dlbar-ios {
+        display: block;
+    }
+}
+
+html.android {
+    #conditional-download-bar #dlbar-nonfx-android {
+        display: block;
+    }
+}
+
+html.firefox-old,
+html.android,
+html.ios {
+    #conditional-download-bar #dlbar-nonfx {
+        display: none;
+    }
+}
+
+
+
+/*--------------------------------------------------------------------------*/
+// Mozilla tab (like tabzilla, but not)
+#moztab {
+    position: absolute;
+    right: 0;
+    top: 0;
```

Comment:
Yeah, that's a little artifact in the image... I can fix it in the sprite, but I can also just nudge it up a pixel in the CSS. Since we know we'll be changing this tab in the semi-near future anyway I opt for the nudge.